scaffold-eth/create-eth
# The graph

### Description : 

### To test 

1. clone repo
2. go to the `the-graph` branch
3. yarn install

Then run:

In first terminal : 

```shell
yarn install && yarn dev
```

In Second terminal : 

```shell
yarn cli 
```

You could choose solidity framework of your choice :), tweaked it a bit to work with both hardhat and foundry 

The third question will prompt for whether to select `the-graph` extension or not. Use space to select / unselect and hit enter. 

> TIP: for project name use `../{anyName}` this will create project outside of this working directory

You could check out generated README.md for instruction's to deploy subgraph locally

^ You will be needing Docker 

I think we could maybe do better on generated README.md's but didn't go too crazy and we can iterate later. (Like accepting requirements, readme heading (eg. SE-2 with Subgraph) etc as args)
